src/components/selections/selectedNominee.scss

```diff
@@ -0,0 +1,4 @@
+
+.ghoul-focused {
+	outline: 1px solid green;
```

Comment:
Can you create a new var to use here called `$focusColor;`

Then set it to `$focusColor` to transparent in the pre-keyboard flows and set to one of our other color variables in each of the rest?